src/components/NewDraftWizard/WizardForms/WizardFillObjectDetailsForm.js

```diff
@@ -245,7 +245,7 @@ const FormContent = ({
 
   const handleClearForm = () => {
     resetTimer()
-    methods.reset({})
+    methods.reset({ undefined })
```

Comment:
This solution seems to work. 
I'm nitpicking and suggesting (don't have to implement though 😄):
```suggestion
    methods.reset(null)
```

Reply:
Yes I did try to put `null` there but it didn't work either. I think it needs to go with a non-empty object inside.
For a new form it totally works fine, the issue only happens when editing an existing form.

Reply:
Alright, I only tested it with new form.